converter/shieldhit/parser.py

```diff
@@ -152,13 +152,17 @@ def _parse_beam(self, json: dict) -> None:
         self.beam_config.particle_name = json["beam"]["particle"].get("name")
         self.beam_config.heavy_ion_a = json["beam"]["particle"]["a"]
         self.beam_config.heavy_ion_z = json["beam"]["particle"]["z"]
-        self.beam_config.energy = json["beam"]["energy"]
-        self.beam_config.energy_spread = json["beam"]["energySpread"]
-        # we use get here to avoid KeyError if the cutoffs are not defined
-        # in that case None will be inserted into the beam config
-        # which is well handled by the converter
-        self.beam_config.energy_low_cutoff = json["beam"].get("energyLowCutoff")
-        self.beam_config.energy_high_cutoff = json["beam"].get("energyHighCutoff")
+
+        energy_unit = json["beam"].get("energyUnit", "MeV")
+        energy_scale_factor = json["beam"]["particle"]["a"] if energy_unit == "MeV" else 1
```

Comment:
Be more pythonic, use something like


```suggestion
        energy_scale_factor = 1
        if energy_unit == "MeV":
            energy_scale_factor = json["beam"]["particle"]["a"]
```